contrib/azurerm/roles/generate-inventory/templates/inventory.j2

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
If I understand, it does 2 things: 
1. create / configure vm / cloud
2. create the inventory


Can we create/generate the inventory  anytime we want without running step 1?